metricbeat/module/windows/perfmon/pdh_windows.go

```diff
@@ -118,20 +133,28 @@ func (q *Query) Execute() error {
 }
 
 type Value struct {
-	Num float64
+	Num interface{}
```

Comment:
From a user perspective I'm not really liking that I have to type assert `Num` to get interpret the value. Maybe in a separate PR we could improve the API for `Query`. One option would be to have separate `FloatValues()` and `Int64Values()` methods that return `map[string]FloatValue` and `map[string]Int64Value` respectively.